### ISSUE TITLE
Test 3 - Tool A02; Name attribute not supported

### ISSUE THREAD

**Issue by lmblackburn** [USDA-NRCS/Wetland-Tools---ArcGIS-Pro#64]
Not sure why this error is being thrown, except that maybe the tool doesn't like corporations?  The producers name was entered as "sprouting farms corporation"
![image](https://user-images.githubusercontent.com/87446239/147978079-8bb18866-f36d-4706-aa1c-ad2f2e33d8d9.png)
I tried to change the producer name and still get the same error.  It seems like it could be tied to the Site_CLU layer.  I also get an error when I try to open the table for that layer.
![image](https://user-images.githubusercontent.com/87446239/147978540-c8aa4b31-b978-42a3-9578-38278bf00ef4.png)

**Comment by lmblackburn** [comment]
Tried to rerun this tool with a different farm and keep getting this error about the attribute, name, not being supported.
![image](https://user-images.githubusercontent.com/87446239/148112455-c45cf5e9-c535-4a61-a377-841828a4d645.png)

**Comment by lmblackburn** [comment]
Decided to move on to step 3, first it asked me to switch to the testing portal, then I get the same 'name' error message as in step 2.
![image](https://user-images.githubusercontent.com/87446239/148113061-536df032-1eca-47b1-b388-0eaaa4be8410.png)

Maybe this is a CLU layer issue?????  Anyone else having problems here?

**Comment by gahisu** [comment]
We have multiple users getting this same error reported above with the production version.

![image](https://user-images.githubusercontent.com/66690345/180056546-865661cd-5040-464d-ad9d-d743bed87b7a.png)

**Comment by J-Hellums** [comment]
We ran into the same issue and used the snippet of python below to identify the layer that was giving us issues. When this was run in the python window it would fail on the layer that was the issue. In our case it was a group layer we pulled from portal. The group name printed, but the layer names inside did not. After removing this and replacing the data everything worked well. 
Good luck 

aprx = arcpy.mp.ArcGISProject("CURRENT")

for m in aprx.listMaps():
                if m.name == "Determinations":
                    for lyr in m.listLayers():
                        print(lyr.name)
